### PR TITLE
Use edgedb-tokio for talking to server

### PR DESCRIPTION
**Pull request by tailhook**

### CODE REVIEW

**Comment by tailhook** [comment]
@fantix, might be quite a big change to review in one chunk. But, please, take a closer look to cloud stuff. And please try running it if you can, because currently its untested mostly.

File: src/portable/create.rs [hunk 1]
```diff
@@ -24,10 +23,11 @@ use crate::print::{self, echo, err_marker, Highlight};
 use crate::process;
 use crate::question;
 
-use edgedb_client::credentials::Credentials;
+use edgedb_tokio::credentials::Credentials;
 
 
-fn ask_name(
+#[tokio::main]
+async fn ask_name(
```
Comment (fantix) [inline src/portable/create.rs:30]: I think this function doesn't have to be async now.

File: src/portable/project.rs [hunk 2]
```diff
@@ -316,7 +315,8 @@ pub fn init(options: &Init, opts: &crate::options::Options) -> anyhow::Result<()
     Ok(())
 }
 
-fn ask_existing_instance_name(
+#[tokio::main]
+async fn ask_existing_instance_name(
```
Comment (fantix) [inline src/portable/project.rs:319]: Ditto, async not needed.

File: src/portable/project.rs [hunk 3]
```diff
@@ -419,7 +419,8 @@ fn do_link(inst: &Handle, options: &Init, stash_dir: &Path)
     })
 }
 
-fn ask_name(
+#[tokio::main]
+async fn ask_name(
```
Comment (fantix) [inline src/portable/project.rs:423]: Ditto, async not required.

**Review by fantix — CHANGES_REQUESTED** [review]
Some cloud operations are crashing because of creating runtime twice in some places, please see inline.

**Review by fantix — APPROVED** [review]

**Pull request merged**